### ISSUE TITLE
Use Destructuring Assignment with the Rest Operator to Reassign Array Elements Run Error

### ISSUE THREAD

**Issue by TomerPacific** [freeCodeCamp/freeCodeCamp#18069]
#### Describe your problem and - if possible - how to reproduce it
In the lesson Use Destructuring Assignment with the Rest Operator to Reassign Array Elements, once the page loads, if you inspect the page itself, you can see there is an error.

#### Add a Link to the page with the problem
https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/es6/use-destructuring-assignment-with-the-rest-operator-to-reassign-array-elements/

#### Tell us about your browser and operating system

* Browser Name: Chrome
* Browser Version: Version 68.0.3440.106 (Official Build) (64-bit)
* Operating System: Windows 7 64 bit

#### If possible, add a screenshot here
![fccerror](https://user-images.githubusercontent.com/23402988/44924627-9137a000-ad54-11e8-8f93-3012fb26f1bb.jpg)

**Comment by TomerPacific** [comment]
@alhazen1 ,
you are correct in your statement, but it is still an error, nevertheless. **If it is such a simple fix, why not do it?**

**Comment by nikrb** [comment]
Can no longer reproduce this error locally on latest master

**Comment by TomerPacific** [comment]
@alhazen1 ,
according to @nikrb statement, I will be closing this issue in 24 hours. If anything changes, reply here.
